### ISSUE TITLE
`traffic_sign=variable_message` has two direction fields

### ISSUE THREAD

**Issue by Dimitar5555** [openstreetmap/id-tagging-schema#1306]
**Describe the bug**
`traffic_sign=variable_message` has two direction fields (one for `direction` and one for `traffic_sign:direction`.

**To Reproduce**
1. Go to https://www.openstreetmap.org/edit#map=19/43.79918/25.91882
2. Select node/12089459546

**Expected behavior**
Only one field should show up. Possibly `direction` since it used on 1.8k elements with `traffic_sign=variable_message` (compared ~1k for `traffic_sign:direction`). 

**Screenshots**
I've entered values for the fields, to show that two tags are applied.
![image](https://github.com/user-attachments/assets/6a7e5cad-03ff-416b-b018-db07ace5550c)

**Comment by tordans** [comment]
Some research on this

The issue is that data/presets/traffic_sign/variable_message.json has two direction fields
1. the explicit field https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/direction_vertex.json
2. the inherited/referenced field from https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/traffic_sign.json which is https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/traffic_sign/direction.json for vertex (https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/direction_point.json for points)

The fields/direction_vertex is there from the start – [History](https://github.com/openstreetmap/id-tagging-schema/commits/ece16f863643169a0e03ad9e1530882e94fe852c/data/fields/direction_vertex.json).
So is the fields/traffic_sign/direction – [History](https://github.com/openstreetmap/id-tagging-schema/commits/main/data/fields/traffic_sign/direction.json)

The fields/direction_vertex is used a lot – [List](https://github.com/search?q=repo%3Aopenstreetmap%2Fid-tagging-schema+direction_vertex+path%3A%2F%5Edata%5C%2Fpresets%5C%2F%2F&type=code)
The fields/traffic_sign/direction is used only once – [List](https://github.com/search?q=repo%3Aopenstreetmap%2Fid-tagging-schema+traffic_sign%2Fdirection+path%3A%2F%5Edata%5C%2Fpresets%5C%2F%2F&type=code)

Given that research, I think the way to solve this is to remove the fields/traffic_sign/direction field and reference fields/direction_vertex for presets/traffic_sign/variable_message.

I will add a label "good first issue" to this.

**Comment by raghav3615** [comment]
@Dimitar5555 open a new PR for this issue. kindly review it

**Comment by tordans** [comment]
Update: See https://github.com/openstreetmap/id-tagging-schema/pull/1351#issuecomment-2933486007 for more information
